cfbs.json

```diff
@@ -830,6 +830,20 @@
         "append enable.cf services/init.cf"
       ]
     },
+    "promise-type-scl-docker_compose": {
+      "description": "Promise type to manage docker compose installation",
+      "tags": ["promise-type", "docker_compose"],
```

Comment:
```suggestion
      "tags": ["promise-type", "experimental"],
```

Reply:
The things I submit are always `experimental`. Is this always for non-cfengine deverlopers ;-)?

Reply:
:rofl:

Reply:
In general, all things are experimental until we've tested / feel confident about making them supported :)